c_emulator/riscv_sim.cpp

```diff
@@ -825,6 +665,8 @@ void run_sail(void)
         goto step_exception;
       flush_logs();
       KILL(sail_int)(&sail_step);
+      if (rvfi)
```

Comment:
I don't think we have a clang-format rule for this yet but I strongly prefer requiring {} for single statement blocks. Less error prone.